datalad/distribution/tests/test_update.py

```diff
@@ -131,6 +131,11 @@ def test_update_fetch_all(src, remote_1, remote_2):
         f.write("different file load")
     rmt2.add("second.txt", git=True, commit=True, msg="Add file to git.")
 
+    # Let's init some special remote which we couldn't really update/fetch
+    if not os.environ.get('DATALAD_TESTS_DATALADREMOTE'):
```

Comment:
Actually, we agreed on not directly using `os.environ` anymore, but use `ConfigManager` to obtain `datalad.tests.dataladremote` instead (should be available via `datalad.api.cfg`, I think). And have this documented in `common_cfg.py`.